data/Application.css

```diff
@@ -0,0 +1,4 @@
+entry.flat {
```

Comment:
From https://github.com/elementary/calculator/issues/275#issuecomment-2636889220:

> That's true, but Gtk.Entry has other classes that still show the focus or active border when the input is focused.

This may be due to the following lines? Isn't there any way to just remove the focused style class and avoid re-implementing the entry.flat class here? :thinking:

https://github.com/elementary/stylesheet/blob/b92637c9608a2455060b89fe17c8be98ffaaf355/src/gtk-4.0/widgets/_entries.scss#L21-L23